common4j/src/main/com/microsoft/identity/common/java/nativeauth/BuildValues.java

```diff
@@ -20,7 +20,9 @@
 //  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
-package com.microsoft.identity.common.java;
+package com.microsoft.identity.common.java.nativeauth;
+
+import com.microsoft.identity.common.java.BuildConfig;
```

Comment:
unnecessary import?

Reply:
no, `BuildConfig` is used on line 35. This has to be imported now due to change in package for `BuildValues` file.